content/en/guides/os-commercial-ecosystem.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Creating an Open Source Commercial Ecosystem
+---
+
+An open source project is successful when it has a thriving ecosystem of companies and products built around it.  So what exactly is an open source commercial ecosystem and how can organizations help create and support sustainable open source projects? There are concrete ways to create confidence in a project’s long-term viability, encourage companies to create commercial products and services on top of open source projects, and re-invest in them with contributions, content, and other resources. These practices for enabling ecosystem development help set your projects up for success.
+
+&nbsp;
+
+**Table of Contents**
+
+- [Defining and creating a  sustainable open source commercial ecosystem](#defining-and-creating-a-sustainable-open-source-commercial-ecosystem)
+- [Ingredients of open source ecosystem success](#ingredients-of-open-source-ecosystem-success)
+- [How to build an open source ecosystem](#how-to-build-an-open-source-ecosystem)
+- [Example: Google Open Source Office](#example-google-open-source-office)
+
+&nbsp;
+
+# Defining and creating a  sustainable open source commercial ecosystem
+
+Open Source Program Offices can interact with open source in two different ways: 
+
+* Outbound processes: contributing to existing or releasing new open source projects
+* Inbound processes: consuming open source projects
+
+
+When companies (and their OSPOs) work on a strategy to build their products and services on open source, they have an incentive to re-invest resources into those projects, which leads to the creation of open source commercial ecosystems. For projects to achieve this level of sustainability, companies must first have enough confidence in a project’s future that they are willing to build commercial dependencies on it.
+
+While there are no guarantees that an open source project will be successful, there are ways to set it up for success through practices that help build commercial ecosystems around it. If you can accelerate this process, it speeds innovation, products get to market faster, and projects see faster adoption and use. It becomes a virtuous cycle as [described by Accel Partners investor Ping Li](https://www.slideshare.net/AccelPartners/the-rise-of-open-adoption-software-oas).
+
+Li has identified three phases of sustainability to focus on Project, Product, and Profit.
+
+* **Project.** Creating a truly useful, purpose-driven project, is step number one in creating a thriving ecosystem.  This can be achieved by creating a great invention from the ground up, by forking an existing invention, or by repurposing an existing project. Did you know that the wildly popular OpenStack cloud platform was originally an in-house open source creation that NASA used? Likewise, the hugely popular Kubernetes orchestration tool evolved from an in-house open source project at Google. Creating, forking and repurposing can all lead to strong demand for projects.
```

Comment:
I'd like to propose to add a statement like "with a strong community" to this section. It is implied and we know this, but it is not explicit stated here and a good technical solution alone is not enough. Moreover, the paragraph after the list states that when those three steps have completed, there is a "robust community" which was not mentioned before.

```suggestion
* **Project.** Creating a truly useful, purpose-driven project with a strong community, is step number one in creating a thriving ecosystem.  This can be achieved by creating a great invention from the ground up, by forking an existing invention, or by repurposing an existing project. Did you know that the wildly popular OpenStack cloud platform was originally an in-house open source creation that NASA used? Likewise, the hugely popular Kubernetes orchestration tool evolved from an in-house open source project at Google. Creating, forking and repurposing can all lead to strong demand for projects.
```